backend/routes/index.js

```diff
@@ -1289,19 +1289,10 @@ module.exports = function() {
     if (!req.isAuthenticated()) {
       res.sendStatus(401)
     } else {
-            
-      let newEvent = {
-        title: req.body.title,
-        start: new Date(req.body.eventstart),
-        end: new Date(req.body.eventend),
-        allDay: req.body.eventallDay,
-        location: req.body.location,
-        description: req.body.description,
-        contacts: req.body.contacts,
-        webpage: req.body.webpage,
-        _startTime: req.body._startTime,
-        _endTime: req.body._endTime,
-      }
+            let newEvent = req.body
+      // let newEvent = {
+      //  ...data
+      // }
```

Comment:
i really like this comment - as a way of explaining what you did 

i guess id say maybe comments usually before the code, and at same indentation